posthog/temporal/data_imports/deltalake_compaction_job.py

```diff
@@ -35,6 +36,10 @@ def trigger_compaction_job(job: ExternalDataJob, schema: ExternalDataSchema) ->
                 ),
             )
         )
+
+        if not DEBUG and not TEST:
```

Comment:
Is there a reason to not do this in the test environment?

Reply:
Yeah we don't run both jobs concurrently in tests. I think it'd be a pain in the ass to set up properly